### PR TITLE
Suppress warnings

### PR DESCRIPTION
**Pull request by aytekaman**
Apply some minor fixes in order to compile project on VC14 with WX flag.

### CODE REVIEW

**Comment by dictoon** [comment]
Interesting, I'm using VC14 as my main compiler (Visual Studio Entreprise 2015, version 14.0.25425.01 Update 3) and I have the /WX switch, yet I'm not getting these warnings...

File: src/appleseed/foundation/meta/tests/test_microfacet.cpp [hunk 1]
```diff
@@ -451,7 +451,7 @@ TEST_SUITE(Foundation_Math_Microfacet)
             for (size_t j = 0; j < num_sample; ++j)
             {
                 static const size_t Bases[] = { 2 };
-                const Vector2f s = hammersley_sequence<float, 2>(Bases, num_sample, i);
+                const Vector2f s = hammersley_sequence<float, 2>(Bases, num_sample, (size_t)i);
```
Comment (dictoon) [inline src/appleseed/foundation/meta/tests/test_microfacet.cpp]: Hm, this looks like a bug actually. Let me ask @gospodnetic and @est77 who worked on this code.

File: src/appleseed/foundation/math/microfacet.cpp [hunk 2]
```diff
@@ -862,7 +862,7 @@ float StdMDF::abgamma(
     const float gm5 = 29944523.0f / 19733142.0f;
     const float gm6 = 109535241009.0f / 48264275462.0f;
 
-    return (0.5f * std::log(2.0 * Pi<float>()) - x + (x - 0.5f) * std::log(x)
+    return (0.5f * std::log(2.0f * Pi<float>()) - x + (x - 0.5f) * std::log(x)
```
Comment (dictoon) [inline src/appleseed/foundation/math/microfacet.cpp]: Oh, I do get this warning now.

File: src/appleseed/foundation/meta/tests/test_microfacet.cpp (hunk 1 above)
Comment (dictoon) [inline src/appleseed/foundation/meta/tests/test_microfacet.cpp]: Confirmed to be a bug, @gospodnetic is currently fixing it. Can you revert these changes?

Comment (aytekaman) [inline src/appleseed/foundation/meta/tests/test_microfacet.cpp]: Ok i will close this PR and open a new one with the other fix.

Comment (aytekaman) [inline src/appleseed/foundation/meta/tests/test_microfacet.cpp]: New PR is here: https://github.com/appleseedhq/appleseed/pull/1340
